src/collection-view.js

```diff
@@ -369,7 +369,7 @@ Marionette.CollectionView = Marionette.View.extend({
 
     // Trigger `attach`
     if (canTriggerAttach && this._triggerAttach) {
-      nestedViews = [view].concat(view._getNestedViews());
+      nestedViews = [view].concat(_.result(view, '_getNestedViews') || []);
```

Comment:
Ahh, I actually copied that from [Region#show](https://github.com/marionettejs/backbone.marionette/blob/4c00dabc3f41a95edac2773824ebecaf6c130986/src/region.js#L155).  It's a not-so-obvious fix for Backbone Views (#2651) that don't have `_getNestedViews`--the `result` call won't fail if that method doesn't exist, and in the non-exist case we get the `[]`.  I do believe `_getNestedViews`, when it does exist, already returns the `[]` default.